### ISSUE TITLE
Template 10 1.1.13p StartUpOrchestrator

### ISSUE THREAD

**Issue by Quico20** [Windows-XAML/Template10#1654]
Jerry,

Im wondering everytime i use the debug location to force a suspended state in my app, it launches the Onnavigatedfromasync to save my app data. Thats ok i understand that well.

What i dong seem to catch is whenever I resume the starkind enumaration always set the Kind as closedbyuser, shouldnt it be set as Suspended???

Whenever its classified like closedbyuser it runs the onstart method at the app.xaml.cs

And it navigates back to the mainpage because im not handling the scenario yet. The point is if ill handle it right now, i would have to set my code to treat closedbyuser as if it were suspended is this the behavior you wanted or am i understanding wrong the lifecycle of template 10 apps?

**Comment by JerryNixon** [comment]
This is Template 10 version 1?

**Comment by Quico20** [comment]
Yes, Jerry Is Template 10 Version 1. I began using T10 V1, and I ended up applying T10 V1 source code directly to me project in order to be able to update my project to Windows 10 April update. 
So far it works great now, I doesn't crash with Microsoft advertising SDK like the nuget version and It doesn't crash with Microsoft Store Services.

The only problem I had besides the one I mentioned here is that I had to sacrifice VS designer, so I do designing most of the time on live while the app is running.

I guess after I finish my app and publish, I'll try to move to T10 V2.

I didn't do great thing with T10 V1. I just updated your source code and got rid of obsolete code, to make it work without errors.
